src/System.Management.Automation/namespaces/SafeRegistryHandle.cs

```diff
@@ -52,13 +46,9 @@ internal SafeRegistryHandle(IntPtr preexistingHandle, bool ownsHandle) : base(ow
 
         protected override bool ReleaseHandle()
         {
-#if UNIX
-            return true;
```

Comment:
`SafeRegistryHandle` is used for transactioned registry, which is not enabled in powershell core. Actually the file `SafeRegistryHandle.cs` should be excluded from building powershell core.